src/features/textEditor/components/TipTapEditor.css

```diff
@@ -1,36 +1,40 @@
-.tiptap-editor menu.menu-bar button {
-  @apply mx-1 my-1 rounded-lg bg-gray-200 px-2 py-1 hover:bg-gray-500 hover:text-white;
+.editor-container {
+  @apply flex h-full w-full flex-col;
+  @apply overflow-hidden rounded-default shadow-default;
 }
 
-.tiptap-editor ul,
-.tiptap-editor ol {
-  @apply ml-7;
+.tiptap-editor cite {
+  @apply not-italic;
 }
 
-.tiptap-editor ul > li {
-  @apply list-disc;
+.tiptap-editor .ProseMirror-focused {
+  outline: none;
 }
 
-.tiptap-editor ol > li {
-  @apply list-decimal;
+.tiptap-editor {
+  @apply flex-1 overflow-y-scroll px-12 py-16;
+  @apply text-lg/6 font-normal;
+  @apply bg-card-bg-primary;
+  @apply flex justify-center;
 }
 
-.tiptap-editor cite {
-  @apply not-italic;
+.tiptap-editor > div {
+  @apply w-full max-w-[45rem];
 }
 
-.tiptap-editor .ProseMirror-focused {
-  outline: none;
+.tiptap-editor h2 {
+  @apply text-2xl font-semibold;
 }
 
-.tiptap-editor {
-  @apply flex-1 overflow-y-scroll pl-1 pr-2 pt-2;
+.tiptap-editor h1 {
+  @apply text-[2rem]/10 font-extrabold;
+  @apply normal-case;
 }
 
 .tiptap-editor *::selection {
   @apply bg-transparent;
 }
 
 .tiptap-editor span.selected {
-  background-color: #c4dcf6;
+  @apply bg-[#c4dcf6];
```

Comment:
Another color to maybe as the design team for a name, right?

Reply:
Yes! I will move it to a variable and ask the design team for which color we should use